rfc-120-csv-preview.md

```diff
@@ -0,0 +1,55 @@
+# CSV Preview
+
+## Summary
+
+rove the 'CSV Preview' feature out of Whitehall and into Government Frontend, changing the structure of the associated URLs so they can also be used with documents published by Content Publisher.
```

Comment:
sorry - pedantic comment of the day `r` should be `M`

Reply:
But I wanted to "rove" it 😆

Reply:
![move_it](https://user-images.githubusercontent.com/647311/78051037-088c4600-7375-11ea-8441-e13f3ad56976.jpg)